### PR TITLE
Fix borked failures check

### PR DESCRIPTION
**Pull request by liath**
Thanks @peterpolman

### CODE REVIEW

**Review by peterpolman — APPROVED** [review]

File: index.js [hunk 1]
```diff
@@ -74,7 +74,7 @@ module.exports = function (options, cb) {
       return taskRunner.runPromisified(params);
     })
     .then((taskDefinition) => {
-      if (taskDefinition.failures) {
+      if (taskDefinition.failures && taskDefinition.failures.length) {
```
Comment (ohookins) [inline index.js:77]: It's an array and it's always present?

**Review by ohookins — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (ohookins) [reply index.js:77]: actually, shouldn't we check that length > 0?

Comment (liath) [reply index.js:77]: That's exactly what we're doing! `0` is falsy. I can switch if you like but I think it's more idiomatic as written.

Comment (liath) [reply index.js:77]: Of course, I golf in js so grain of salt wrt my opinions of what's legible lol

Comment (ohookins) [reply index.js:77]: I think `> 0` is clearer as to the intent, even if it's redundant.

**Review by pdasilva — APPROVED** [review]

**Pull request merged**